### PR TITLE
Step 1.5, deploy the project to Firebase hosting 

### PR DESCRIPTION
**Pull request by mohammad-suliman**
This PR deploys the project to Firebase hosting by adding the required files to this project. Note also that the ID attribute of the process button was removed as agreed with Ofer.

### CODE REVIEW

File: projects/sensory_interface/public/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```
Comment (oferb) [inline projects/sensory_interface/public/index.html]: Why do you need `index.html` twice?
If you need it in `public`, you can remove it from the parent folder. Same for `app.js`.

File: projects/sensory_interface/index.html [hunk 2]
```diff
@@ -9,9 +9,18 @@
 <body>
     <textarea id="textInput">1	2	3
 4	5	6</textarea>
-    <button id=" processButton" onclick="processData()">process </button>
+    <button onclick="processData()">process </button>
     <br>
     <div id="tableContainer"></div>
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
```
Comment (oferb) [inline projects/sensory_interface/index.html]: Right now you're using only Firebase hosting, and I don't think you need the scripts for that.
Try removing them and deploying and see if it still works.

File: projects/sensory_interface/index.html [hunk 3]
```diff
@@ -9,9 +9,18 @@
 <body>
     <textarea id="textInput">1	2	3
 4	5	6</textarea>
-    <button id=" processButton" onclick="processData()">process </button>
```
Comment (oferb) [inline projects/sensory_interface/index.html]: Good. ID is only needed if you want to obtain the element by ID, or reference it by ID from the CSS.

File: projects/sensory_interface/index.html (hunk 2 above)
Comment (mohammad-suliman) [reply projects/sensory_interface/index.html]: it works without the scripts as you suggested. Removing...

Comment (oferb) [reply projects/sensory_interface/index.html]: Great :)

**Pull request merged**